code/game/jobs/job/assistant.dm

```diff
@@ -28,6 +28,10 @@
 	H.equip_or_collect(new /obj/item/clothing/under/color/grey(H), slot_w_uniform)
 	H.equip_or_collect(new /obj/item/clothing/shoes/black(H), slot_shoes)
 	H.equip_or_collect(new H.species.survival_gear(H.back), slot_in_backpack)
+	if(prob(1))
+		H.equip_or_collect(new /obj/item/weapon/storage/bag/plasticbag(H), slot_head)
```

Comment:
If you can't learn the interface within one minute of joining you don't deserve to stay on this server

casuals please leave